lib/run_cucumber.js

```diff
@@ -20,21 +18,36 @@ const cucumber_api = require('@cucumber/cucumber/api');
     "paths": ["docassemble/*/data/sources/*.feature"],
     "retry": 1,
   }
-  if (just_args.length > 0) {
+
+
+  // If they're running it from an interview on their server instead of from GitHub
+  if ( session_vars.get_origin() === `playground` ) {
+    provided_config.paths.push(`/usr/share/docassemble/files/playgroundsources/${session_vars.get_user_id()}/${session_vars.get_user_project_name()}/*.feature`);
+    // For S3
+    provided_config.paths.push(`/tmp/playgroundsources/${session_vars.get_user_id()}/${session_vars.get_user_project_name()}/*.feature`);
```

Comment:
This is fine for now, but I didn't see an issue anywhere, so I wanted to note; I think the next refactor here should be do add a proper command line argument handler like [minimist](https://github.com/minimistjs/minimist), and then pass these two paths through the command line instead. It has a few benefits:
* removes the necessity of 2 additional environment variables as well, `get_user_id`, and `get_user_project_name`
* puts docassemble specific knowledge (what these specific paths are) in the docassemble specific package (ALKilnInThePlayground)
* makes it more flexible in general; I made a quick little tool that could run 10 different versions of Kiln at the same time by running them in different working directories and pointing them to the source directory elsewhere. 

I did a little preliminary work on this in https://github.com/SuffolkLITLab/ALKiln/commit/f8e9921788abc3c45748a8090e451275081145e9, but I'm trying to shift priorities for a bit, and don't with this to be slowed down by me.

Reply:
Makes a ton of sense! #687